_examples/producer/producer.go

```diff
@@ -6,12 +6,13 @@ package main
 import (
 	"context"
 	"flag"
-	amqp "github.com/rabbitmq/amqp091-go"
 	"log"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
+
+	amqp "github.com/rabbitmq/amqp091-go"
```

Comment:
Was this change due to your preference, or to a tool auto-formatting the code?

Reply:
by go fmt

Reply:
Is there other example code that should be formatted as part of this PR?

Reply:
`go fmt ./...` doesn't make this change for me 😕 Are you running `go fmt` with any special flag?

Reply:
> `go fmt ./...` doesn't make this change for me 😕 Are you running `go fmt` with any special flag?

Yes, you are right, I think my submitted code was automatically formatted by the development tool (goland); I hope you ignore the submission about here. Thanks